apps/web/src/components/layout/constants.ts

```diff
@@ -1 +1 @@
-export const HEADER_HEIGHT = '40px';
+export const HEADER_HEIGHT = 40;
```

Comment:
converted to num for easy addition/substraction with other height values in layouts while using `calc`